examples/enable/shapes/run.py

```diff
@@ -1,12 +1,12 @@
 """ An example showing moveable shapes. """
-# Enthought library imports.
-from enable.api import Container, Window
-from enable.example_support import DemoFrame, demo_main
-
 # Local imports
 from box import Box
 from circle import Circle
 
+# Enthought library imports.
+from enable.api import Container, Window
+from enable.example_support import DemoFrame, demo_main
+
 
 class MyFrame(DemoFrame):
```

Comment:
This still uses `DemoFrame`. Can be fixed by refactor suggested by @corranwebster.